extensions/wikia/Search/classes/Test/Controller/SearchControllerTest.php

```diff
@@ -570,150 +443,83 @@ public function testPaginationWithConfigNoResults2() {
-			->will		( $this->returnValue( 10 ) )
-		;
+			->expects( $this->once() )
+			->method( 'getQuery' )
+			->will( $this->returnValue( $mockQuery ) );
+
 		$mockConfig
-			->expects	( $this->at( $incr++ ) )
-			->method	( 'getInterWiki' )
-			->will		( $this->returnValue( false ) )
-		;
+			->expects( $this->once() )
+			->method( 'getInterWiki' )
+			->will( $this->returnValue( false ) );
+
 		$mockConfig
-			->expects	( $this->at( $incr++ ) )
-			->method	( 'getResultsFound' )
-			->will		( $this->returnValue( 200 ) )
-		;
+			->expects( $this->once() )
+			->method( 'getResultsFound' )
+			->will( $this->returnValue( 200 ) );
+
 		$mockConfig
-			->expects	( $this->at( $incr++ ) )
-			->method	( 'getNamespaces' )
-			->will		( $this->returnValue( array( NS_MAIN ) ) )
-		;
+			->expects( $this->once() )
+			->method( 'getNamespaces' )
+			->will( $this->returnValue( [ NS_MAIN ] ) );
+
 		$mockConfig
-			->expects	( $this->at( $incr++ ) )
-			->method	( 'getLimit' )
-			->will		( $this->returnValue( 20 ) )
-		;
+			->expects( $this->once() )
+			->method( 'getLimit' )
+			->will( $this->returnValue( 20 ) );
+
 		$mockConfig
-			->expects	( $this->at( $incr++ ) )
-			->method	( 'getPublicFilterKeys' )
-			->will		( $this->returnValue( array( 'is_image' ) ) )
-		;
-		$incr2 = 1;
-		$mockController
-			->expects	( $this->at( $incr2++ ) )
-			->method	( 'setVal' )
-			->with		( 'query', 'foo' )
-		;
-		$mockController
-			->expects	( $this->at( $incr2++ ) )
-			->method	( 'setVal' )
-			->with		( 'pagesNum', '10' )
-		;
-		$mockController
-			->expects	( $this->at( $incr2++ ) )
-			->method	( 'setVal' )
-			->with		( 'currentPage', 2 )
-		;
-		$mockController
-			->expects	( $this->at( $incr2++ ) )
-			->method	( 'setVal' )
-			->with		( 'windowFirstPage', 1 )
-		;
-		$mockController
-			->expects	( $this->at( $incr2++ ) )
-			->method	( 'setVal' )
-			->with		( 'windowLastPage', 7 )
-		;
-		$mockController
-			->expects	( $this->at( $incr2++ ) )
-			->method	( 'setVal' )
-			->with		( 'pageTitle', $mockTitle )
-		;
-		$mockController
-			->expects	( $this->at( $incr2++ ) )
-			->method	( 'setVal' )
-			->with		( 'crossWikia', false )
-		;
-		$mockController
-			->expects	( $this->at( $incr2++ ) )
-			->method	( 'setVal' )
-			->with		( 'resultsCount', 200 )
-		;
-		$mockController
-			->expects	( $this->at( $incr2++ ) )
-			->method	( 'setVal' )
-			->with		( 'namespaces', array( NS_MAIN ) )
-		;
-		$mockController
-			->expects	( $this->at( $incr2++ ) )
-			->method	( 'setVal' )
-			->with		( 'limit', 20 )
-		;
-		$mockController
-			->expects	( $this->at( $incr2++ ) )
-			->method	( 'setVal' )
-			->with		( 'filters', array( 'is_image' ) )
-		;
-		$mockController
-			->expects	( $this->at( $incr2++ ) )
-			->method	( 'getVal' )
-			->with		( 'by_category', false )
-			->will		( $this->returnValue( false ) )
-		;
+			->expects( $this->once() )
+			->method( 'getPublicFilterKeys' )
+			->will( $this->returnValue( [ 'is_image' ] ) );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "is_image" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)